packages/cli/bin/cli.js

```diff
@@ -74,6 +81,17 @@ const argv = yargs
     describe: 'set log level to debug',
     type: 'boolean',
   })
+  .check(argv => {
+    if (getIsInProject(argv.src)) {
```

Comment:
Some project commands support a `path` argument, which means that they can run from outside of a project folder. Example `hs project upload ./path/to/my/project`

I don't think this check will catch that use case. Does `argv` also include info about the options passed in?

Reply:
Discussed this yesterday in the 1-1, and we're going to hold off on covering the scenario with the `path` argument for now, since it seems much less common.